selection_tools/prepare_histograms.C

```diff
@@ -50,39 +50,72 @@ int main(int argc, char *argv[])
     {"muon_pt",  new TH1D("muon_pt", "muon_pt", 1000, 0, 1000)},
     {"muon_pz",  new TH1D("muon_pz", "muon_pz", 1000, -1000, 1000)},
     {"muon_mass",  new TH1D("muon_mass", "muon_mass", 1000, 0, 10)},
+    {"muon_eta",  new TH1D("muon_eta", "muon_eta", 1000, -100, 100)},
+    {"muon_phi",  new TH1D("muon_phi", "muon_phi", 1000, -100, 100)},
+    {"muon_y",  new TH1D("muon_y", "muon_y", 1000, -100, 100)},
+    {"muon_theta",  new TH1D("muon_theta", "muon_theta", 1000, -100, 100)},
+    {"muon_lxy",  new TH1D("muon_lxy", "muon_lxy", 1000, 0, 100)},
+    {"muon_lz",  new TH1D("muon_lz", "muon_lz", 1000, 0, 100)},
+    {"muon_lxyz",  new TH1D("muon_lxyz", "muon_lxyz", 1000, 0, 100)},
+    {"muon_ctau",  new TH1D("muon_ctau", "muon_ctau", 1000, 0, 100)},
+    {"muon_boost",  new TH1D("muon_boost", "muon_boost", 1000, 0, 1000)},
+
+    {"muon_deltaR",  new TH1D("muon_deltaR", "muon_deltaR", 1000, 0, 100)},
+    {"muon_deltaPhi",  new TH1D("muon_deltaPhi", "muon_deltaPhi", 1000, -100, 100)},
     
     {"dimuon_pt",  new TH1D("dimuon_pt", "dimuon_pt", 1000, 0, 1000)},
     {"dimuon_pz",  new TH1D("dimuon_pz", "dimuon_pz", 1000, -1000, 1000)},
     {"dimuon_mass",  new TH1D("dimuon_mass", "dimuon_mass", 1000, 0, 100)},
+    {"dimuon_eta",  new TH1D("dimuon_eta", "dimuon_eta", 1000, -100, 100)},
+    {"dimuon_phi",  new TH1D("dimuon_phi", "dimuon_phi", 1000, -100, 100)},
     
     {"first_mother_pt",  new TH1D("first_mother_pt", "first_mother_pt", 1000, 0, 1000)},
     {"first_mother_pz",  new TH1D("first_mother_pz", "first_mother_pz", 1000, -1000, 1000)},
     {"first_mother_mass",  new TH1D("first_mother_mass", "first_mother_mass", 1000, 0, 200)},
+    {"first_mother_eta",  new TH1D("first_mother_eta", "first_mother_eta", 1000, -100, 100)},
+    {"first_mother_phi",  new TH1D("first_mother_phi", "first_mother_phi", 1000, -100, 100)},
     {"first_mother_lxy",  new TH1D("first_mother_lxy", "first_mother_lxy", 1000, 0, 100)},
     {"first_mother_lz",  new TH1D("first_mother_lz", "first_mother_lz", 1000, 0, 100)},
     {"first_mother_lxyz",  new TH1D("first_mother_lxyz", "first_mother_lxyz", 1000, 0, 100)},
     {"first_mother_ctau",  new TH1D("first_mother_ctau", "first_mother_ctau", 1000, 0, 100)},
     {"first_mother_boost",  new TH1D("first_mother_boost", "first_mother_boost", 1000, 0, 100)},
```

Comment:
Did you consider to change the binning for some of those histograms? I remember we discussed that in some cases what we have here didn't make much sense. Could you have another look and decide if we should add more bins or extend some ranges?